ssas/README.md

```diff
@@ -4,76 +4,66 @@ The SSAS can be run as a standalone web service or embedded as a library.
 
 # Code Organization
 
-The service package contains a standalone http service that encapsulates the authorization library.
+The outline below shows the physical directory structure of the code, with package names highlighted. The service package contains a standalone http service that presents the authorization library via two http servers, one for admin tasks and one for authorization tasks.
 
-Imports always go up the directory tree from leaves; that is, parents do not import from their children. Children may import from their siblings. In order to maintain the service and library distinction, the ssas package and packages in plugins must not import from packages in the service directory.
-
-The outline below shows physical the directory structure of the code, with package names highlighted. The service and plugins directories are used for organization, but are not packages themselves. The main package is located in the service directory. Currently, the go files listed are the intended locations of code that will be migrated from bcda auth when an in-progress refactor is completed. The go files currently in place are placeholders that served to set up the code tree and illustrate the intended relationship between the service (main.go) and the ssas.
-
-Not shown: _test files for every .go file are assumed to be present parallel to their test subject.
+Imports always go up the directory tree from leaves; that is, parents do not import from their children. Children may import from their siblings. In short, the ssas package and okta packages must not import from packages in the service directory.
```

Comment:
```suggestion
Imports always go up the directory tree from leaves; that is, parents do not import from their children. Children may import from their siblings. In short, the `ssas` and `okta` packages must not import from packages in the service directory.
```
Wording seemed to imply that there was one `ssas` package but multiple `okta` packages (or maybe that's the case and I missed it)